src/components/progress/progress.stories.tsx

```diff
@@ -17,33 +21,43 @@ const meta: Meta<typeof Progress> = {
 type Story = StoryObj<typeof Progress>;
 
 /**
- * Default usage example of the Progress component.
+ * Default example of the Progress component.
  */
 export const Default: Story = {
-    args: {
-        value: 10,
+    render: (props) => {
+        return <Progress {...props} />;
```

Comment:
Following the above suggestion:
```suggestion
    args: {
        value: 50,
```